code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -176,6 +193,13 @@
 	. = ..()
 	if(.)
 		weapon_weight = WEAPON_MEDIUM
+		inhand_icon_state = "gun"
+		worn_icon_state = "gun"
+		worn_icon = 'icons/mob/clothing/belt.dmi'
+		lefthand_file = 'icons/mob/inhands/weapons/guns_lefthand.dmi'
+		righthand_file = 'icons/mob/inhands/weapons/guns_righthand.dmi'
```

Comment:
should be handled by the base proc.

Reply:
Reverted. Updated the base proc for worn_icon_state. Omitted worn_icon and inhands files.